frontend/pages/recipe/_slug/ocr-editor.vue

```diff
@@ -0,0 +1,922 @@
+            <div class="d-flex justify-end mt-2">
+              <RecipeDialogBulkAdd class="ml-1 mr-1" :input-text-prop="selectedText" @bulk-data="addStep" />
+              <BaseButton @click="addStep()"> {{ $t("general.new") }}</BaseButton>
+            </div>
+            <RecipeInstructions
+              v-model="recipe.recipeInstructions"
+              :ingredients="recipe.recipeIngredient"
+              :disable-amount="recipe.settings.disableAmount"
+              :edit="true"
+              :recipe-id="recipe.id"
+              :recipe-slug="recipe.slug"
+              :assets.sync="recipe.assets"
+              @clickInstructionField="setSingleStep"
+            />
+          </v-tab-item>
+        </v-tabs-items>
+      </v-col>
+    </v-row>
+    <v-dialog v-model="showHelp" width="800px">
+      <v-card>
+        <v-app-bar dense dark color="primary" class="mb-2">
+          <v-icon large left>
+            {{ $globals.icons.help }}
+          </v-icon>
+          <v-toolbar-title class="headline"> Help </v-toolbar-title>
+          <v-spacer></v-spacer>
+        </v-app-bar>
+        <v-card-text>
+          <h1>Mouse modes</h1>
+          <v-divider class="mb-2 mt-1" />
+          <h2 class="my-2">
+            <v-icon> {{ $globals.icons.selectMode }} </v-icon>{{ $t("ocr-editor.help-dialog.selection-mode") }}
+          </h2>
+          <p class="my-1">{{ $t("ocr-editor.help-dialog.selection-mode") }}</p>
+          <ol>
+            <li>{{ $t("ocr-editor.help-dialog.selection-mode-steps.draw") }}</li>
+            <li>{{ $t("ocr-editor.help-dialog.selection-mode-steps.click") }}</li>
+            <li>{{ $t("ocr-editor.help-dialog.selection-mode-steps.result") }}</li>
+          </ol>
+          <h2 class="my-2">
+            <v-icon> {{ $globals.icons.panAndZoom }} </v-icon>{{ $t("ocr-editor.help-dialog.pan-and-zoom-mode") }}
+          </h2>
+          {{ $t("ocr-editor.help-dialog.pan-and-zoom-desc") }}
+          <h1 class="mt-5">{{ $t("ocr-editor.help-dialog.split-text-mode") }}</h1>
+          <v-divider class="mb-2 mt-1" />
+          <h2 class="my-2">
+            <v-icon> {{ $globals.icons.preserveLines }} </v-icon
+            >{{ $t("ocr-editor.help-dialog.split-modes.line-mode") }}
+          </h2>
+          <p>
+            {{ $t("ocr-editor.help-dialog.split-modes.line-mode-desc") }}
+          </p>
+          <h2 class="my-2">
+            <v-icon> {{ $globals.icons.preserveBlocks }} </v-icon>
+            {{ $t("ocr-editor.help-dialog.split-modes.block-mode") }}
+          </h2>
+          <p>
+            {{ $t("ocr-editor.help-dialog.split-modes.block-mode-desc") }}
+          </p>
+          <h2 class="my-2">
+            <v-icon> {{ $globals.icons.flatten }} </v-icon> {{ $t("ocr-editor.help-dialog.split-modes.flat-mode") }}
+          </h2>
+          <p>{{ $t("ocr-editor.help-dialog.split-modes.flat-mode-desc") }}</p>
+        </v-card-text>
+      </v-card>
+    </v-dialog>
```

Comment:
Can we just show this at the bottom or top of the page? I think that would prevent a lot of Github Issues